e2e/tests/api-driven/src/invite-to-pay/mocks/session.json

```diff
@@ -280,7 +279,7 @@
               "designated.nationalPark",
               "designated.nationalPark.broads",
               "designated.WHS",
-              "designated.SPA",
```

Comment:
This session mock data was originally generated on staging. I've updatd it's `lowcal_session.data` using these changes (even though post-sanitation) to confirm that it validates successfully against ODP schema: https://api.editor.planx.dev/admin/session/d4d9249a-dc1e-4996-9bfc-15fd3adab242/digital-planning-application